### PR TITLE
move static timeout math ops to setup() function

### PR DESCRIPTION
**Pull request by csurf**
These timeout values don't change during main loop, so they can be moved to setup function and calculated only once, after bind data has been received. This uses 134 more bytes for variable allocation, but avoids extra cycles during main loop execution for 32-bit math. Is this OK?

Compiled & tested, works fine...

### CODE REVIEW

File: RX.h [hunk 1]
```diff
@@ -9,6 +9,11 @@ uint32_t lastPacketTimeUs = 0;
 uint32_t lastRSSITimeUs = 0;
 uint32_t linkLossTimeMs;
 
+uint32_t hopInterval = 0;
+uint32_t hopTimeout = 0;
+uint32_t hopTimeoutSlow = 0;
+uint32_t RSSI_timeout = 0;
```
Comment (mikeller) [inline RX.h]: Just to be consistent, can you change this to camel case?

Comment (csurf) [inline RX.h]: isn't it already in CC? Do you mean capitalizing the first letter? 

For RSSI_timeout, I believe I copied the format from some other 'RSSI_*' variable that I saw somewhere in the code (I'd have to go back and check, can't remember right now).

Comment (mikeller) [inline RX.h]: My hunch would be to write it `rssiTimeout`, which seems to be most common way to deal with abbreviations. In general, I don't like the mix of camel case and underscores, as it makes remembering what uses camel case and / or underscores hard.

Comment (csurf) [inline RX.h]: ok, I'm cool with that...
yeah, I don't personally like doing the underscore thing, but I was trying to emulate what I had seen elsewhere in kha's code.

I'll try to get it fixed&pushed later today.

**Pull request merged**